README.md

```diff
@@ -14,4 +14,4 @@ https://www.loginradius.com/docs/developer
 - `npm start` in this directory
 -  Open a browser to the link provided in the console.
 
-
+**Note:** Before start contributing to this docs you must read [contributing guidelines](CONTRIBUTING.md).
```

Comment:
I think the small letter `md` would not link to the file. I was tried to click the link in display mode and it bring me to the 404 page.

https://github.com/LoginRadius/docs/blob/master/CONTRIBUTING.MD

```suggestion
**Note:** Before start contributing to this docs you must read [contributing guidelines](CONTRIBUTING.MD).
```

Reply:
Thanks for letting me know on this. I just updated my PR kindly check now.